src/annotator/config/test/settings-test.js

```diff
@@ -22,7 +22,90 @@ describe('annotator.config.settingsFrom', function () {
     $imports.$restore();
   });
 
-  describe('#sidebarAppUrl', function () {
+  describe('#notebookAppUrl', () => {
+    function appendNotebookLinkToDocument(href) {
+      const link = document.createElement('link');
+      link.type = 'application/annotator+html';
+      link.rel = 'notebook';
+      if (href) {
+        link.href = href;
+      }
+      document.head.appendChild(link);
+      return link;
+    }
+
+    context("when there's an application/annotator+html notebook link", () => {
+      let link;
+
+      beforeEach('add an application/annotator+html notebook <link>', () => {
+        link = appendNotebookLinkToDocument('http://example.com/app.html');
+      });
+
+      afterEach('tidy up the notebook link', () => {
+        document.head.removeChild(link);
```

Comment:
```suggestion
        link.remove();
```

This is a bit shorter and doesn't need to be updated if you decide to move the element to a different parent in future.

Reply:
Done throughout these tests.